web/src/scheduled_messages_ui.js

```diff
@@ -102,3 +105,54 @@ export function initialize() {
         e.stopPropagation();
     });
 }
+
+export function should_show_schedule_message_indicator() {
+    const scheduled_messages_list = Object.values(scheduled_messages.scheduled_messages_data);
+    const filter = narrow_state.filter();
+    const is_conversation_view = filter === undefined ? false : filter.is_conversation_view();
+    const current_narrow = narrow_state.narrowed_to_pms() ? "private" : "stream";
```

Comment:
Let's define this after the `is_conversation_view` check on the next line, and call it `current_view_type`.

A `narrow` I would expect to be some more complex type than a string.